crates/harness/src/monolith.rs

```diff
@@ -45,7 +49,7 @@ pub struct Monolith {
 /// This is necessary because the monolith is split into two tasks: one for the websocket
 /// connection and one for the HTTP mock server.
 #[derive(Debug, Default)]
-pub(crate) struct MonolithState {
```

Comment:
I know this line was removed but what's the difference between pub(crate) struct and pub struct?

Reply:
`pub(crate)` makes it visible within the entire crate. `pub` makes it visible to external crates.